src/components/contacts/AddRandom.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import contacts from '../../data/contacts.json'
+
+class AddRandomContact extends React.Component{
```

Comment:
Very good job mate! You went one step ahead creating components! Everything working properly. 👍🏻